cea/utilities/solar_equations.py

```diff
@@ -306,18 +307,95 @@ def optimal_angle_and_tilt(sensors_metadata_clean, latitude, solar_properties, m
         same as the roof. Sensors on flat roofs are all south facing.
     """
     # calculate panel tilt angle (B) for flat roofs (tilt < 5 degrees), slope roofs and walls.
-    optimal_angle_flat_deg = calc_optimal_angle(180, latitude,
+    input_angle_rad = radians(panel_tilt_angle)
+    sensors_metadata_clean['tilt_deg'] = np.vectorize(acos)(sensors_metadata_clean['Zdir'])  # surface tilt angle in rad
```

Comment:
is this line redundant?

Reply:
No, it calculates the tilt angle in radians, which is converted to degrees in the line after.